src/test/java/io/xpring/xrpl/legacy/LegacyDefaultXpringClientTest.java

```diff
@@ -26,9 +26,9 @@
  */
 class GRPCResult<T> {
```

Comment:
Just noting here that if this code wasn't going away, we should totally de-dupe this class across the tests. I was lazy when I did split the protocol buffers which is why this is this way.